### PR TITLE
Feature/main/upgrade demo app rn

### PR DESCRIPTION
**Pull request by NazarYablonskiy**
### Demo app:
upgrade RN to 0.61.4
upgrade idnow sdk ios to 3.22.0

### CODE REVIEW

File: demo/ReactNativeIDnowSample/ios/Podfile.lock [hunk 1]
```diff
@@ -14,37 +14,375 @@ PODS:
...
+    - boost-for-react-native
+    - DoubleConversion
+    - glog
+  - glog (0.3.5)
+  - IDnowSDK (3.22.0):
     - AFNetworking (~> 3.2.1)
     - FLAnimatedImage (~> 1.0)
     - libPhoneNumber-iOS (~> 0.9)
     - Masonry (~> 1.1.0)
+    - Sentry (~> 4.3.4)
```
Comment (arifaydogmus) [inline demo/ReactNativeIDnowSample/ios/Podfile.lock]: You already removed Sentry. Could you please update the Podlock?

Comment (NazarYablonskiy) [reply demo/ReactNativeIDnowSample/ios/Podfile.lock]: @arifaydogmus which lock file?

Comment (NazarYablonskiy) [reply demo/ReactNativeIDnowSample/ios/Podfile.lock]: @arifaydogmus which lock file?

Comment (NazarYablonskiy) [reply demo/ReactNativeIDnowSample/ios/Podfile.lock]: @arifaydogmus you probably missed that `Sentry (~> 4.3.4)` is installed by `IDnowSDK (3.22.0)`

Comment (arifaydogmus) [reply demo/ReactNativeIDnowSample/ios/Podfile.lock]: My bad. 🙏  You're right.

**Pull request merged**